packages/next/server/render.tsx

```diff
@@ -1378,7 +1403,7 @@ export async function renderToHTML(
   )
 
   let documentHTML: string
-  if (concurrentFeatures) {
+  if (hasConcurrentFeatures) {
```

Comment:
FYI: I'm going to change this slightly in an upcoming PR. Since React 18 _always_ uses the Fizz renderer, I'm going to make it so that `reactRoot: true` _always` uses `renderToStream` to render, but without streaming.

Reply:
Sounds great and looking forward to seeing your PR! This PR is more like keeping the current behavior unchanged.

Reply:
Yup, this just seemed like a convenient place to let you know 😂